main/glossary/README.md

```diff
@@ -67,151 +102,362 @@ synchronously compared for structural equivalence.
+
+## Petname
+
+Petnames are your personal names for objects. No one else can see or modify a petname without your permission. 
+Think of them as similar to a phone's contacts list. The actual phone number is what a phone uses to call 
+someone, but to more easily tell who a number is associated with, it's assigned a petname, such 
+as Mom, Grandpa, Kate S., etc. In the Agoric platform, petnames are used in [wallets](#wallet).
+
+## Presence 
+A local version of a remote object that serves as the remote object's proxy. 
+If `obj` is a presence of a remote object, you can send messages to the remote object by using `E()` on `obj`. 
+For more information, see the [JavaScript Distributed Programming Guide](/distributed-programming.md). 
+
+## Proposal
+
+Proposals are records with `give`, `want`, and `exit` keys. [Offers](#offer) must include a proposal, which states
+what asset you want, what asset you will give for it, and how/when the offer maker can cancel the offer
+(see [Exit Rule](#exit-rule) for details on the last). For example:
+```
+const myProposal = harden({
+  give: { Asset: quatloosAmountMath.make(4)},
+  want: { Price: moolaAmountMath.make(15) },
+  exit: { 'onDemand'
+})
+```
+`give` and `want` use [keywords](#keywords) defined by the contract. Each specifies via an [amount](#amount), a description of what
+asset they are willing to give/want to get, and how much of it. 
+
+## Purse 
+Purses hold [amounts](#amount) of a certain [mint](#mint) issued assets. Specifically amounts that are _stationary_.
+Purses can transfer part of their held balance to a [payment](#payment), which is usually used to transfer value. 
+A purse's contents are all of the same [brand](#brand).
+
+For more information, see the [ERTP Guide's Purses section](/ertp/guide/purses-and-payments.md#purses-and-payments) and the
+[ERTP API's Purses section](/ertp/api/purse.md).
+
+## Quatloos
+An imaginary currency Agoric documentation uses in examples. For its origins, see the Wikipedia entry for the Star Trek 
+episode [The Gamesters of Triskelion](https://en.wikipedia.org/wiki/The_Gamesters_of_Triskelion).
+
+## Reallocate/Reallocation
+
+When an [offer](#offer) exits due either to success or rejection, the associated [payments](#payment) that were [escrowed](#escrow) with Zoe are automatically
+appropriately reallocated to the offer participants. If the offer was rejected or otherwise failed, Zoe gives the offer-making party back what
+they escrowed. If the offer was accepted, Zoe allocates to each party what they said they wanted. There are cases where the reallocation both
+gives a party what they wanted and some of what they escrowed. For example, in an auction a party might have escrowed 10 Quatloos to make their
+highest bid if necessary, but they won the item with a bid of just 8 Quatloos. The reallocation would give them both the item they won and
+a refund of the 2 Quatloos that weren't needed to purchase it. 
```

Comment:
No, this is very confused. A reallocation is a transfer of amounts between seats within Zoe, a change in allocation. When a seat exits, the seat gets the allocation it currently has, as a payout. Exiting *never* triggers a reallocation. Exiting has nothing to do with reallocation. I think you are talking about allocations here.